Solutions/BitSight/Analytic Rules/BitSightDropInHeadlineRating.yaml

```diff
@@ -0,0 +1,36 @@
+id: b11fdc35-6368-4cc0-8128-52cd2e2cdda0
+name: BitSight - drop in the headline rating
+description: |
+  'Rule helps to detect if headline ratings is drop in BitSight.'
+severity: High
+status: Available
+requiredDataConnectors:
+  - connectorId: BitSight
+    dataTypes:
+      - BitSightGraphData
+queryFrequency: 1d
+queryPeriod: 24h
+triggerOperator: GreaterThan
+triggerThreshold: 0
+tactics:
+  - Command And Control
+query: |
+    let timeframe = 24h;
+    BitSightGraphData
+    | where ingestion_time() > ago(timeframe)
+    | where toint(RatingDifferance) < 0
+    | project RatingDate, Rating, CompanyName, RatingDifferance
+incidentConfiguration:
+    createIncident: true
+eventGroupingSettings:
+    aggregationKind: AlertPerResult
+alertDetailsOverride:
+    alertDisplayNameFormat: 'BitSight : Alert for drop in the headline rating of {{CompanyName}}.'
+    alertDescriptionFormat: 'Alert is generated for {{CompanyName}}.\n\nRating Date: {{RatingDate}}\nRating Drop: {{RatingDifferance}}'
+entityMappings:
+  - entityType: Sentinel entities
+    fieldMappings:
+      - identifier: Entities
+        columnName: AlertId
```

Comment:
AlertId is not project and will not get populated.
Same is applicable to all the queries where we are referring to this field.